### PR TITLE
Create common abis folder

### PR DESCRIPTION
**Pull request by mdantonio**
To avoid duplications and collisions, we want to merge all abis
definitions into a single shared folder. To ease the access to the abi
json files from any location, added a common helper to obtain both the
path and the json content

### CODE REVIEW

File: thetanuts/README.md [hunk 1]
```diff
@@ -5,11 +5,12 @@ Python SDK to interact with Thetanuts Finance auctions on Paradigm
 ## Setup to run sample thetanuts test script
 
 Open a container with python
-$ docker run -it --rm \
-    -v $(pwd):/tmp/code -w /tmp/code \
-    -p 8888:8888 \
-    python:3.10 \
-    bash
+
+    $ docker run -it --rm \
```
Comment (pdonorio) [inline thetanuts/README.md:9]: Why this change?

File: sdk_commons/pyproject.toml [hunk 2]
```diff
@@ -12,6 +12,7 @@ authors = [
 ]
 packages = [
     { include = 'sdk_commons' },
+    { include = 'sdk_commons/abis' },
```
Comment (pdonorio) [inline sdk_commons/pyproject.toml:15]: Isn't this supposed to be `'abis'` only?

Comment (mdantonio) [reply sdk_commons/pyproject.toml:15]: No, in the end I put abis into sdk_commons to access the files easier from the get_abi helper

File: thetanuts/README.md (hunk 1 above)
Comment (mdantonio) [reply thetanuts/README.md:9]: Wasn't formatted as code, indented to format properly.
Unrelated with the main topic of this PR, but it's a so small change that does not worth a dedicated PR to fix it

Comment (pdonorio) [reply thetanuts/README.md:9]: Ok, just checking if it was intended.

**Review by pdonorio — APPROVED** [review]
